src/task/ReceiveTURNCredentialsTask.ts

```diff
@@ -104,5 +106,7 @@ export default class ReceiveTURNCredentialsTask extends BaseTask {
         return !!uri;
       });
     this.context.turnCredentials.username = responseBodyJson.username;
+    this.context.logger.info(`TURN URIs: ${this.context.turnCredentials.uris}`);
+    this.context.logger.info(`Username: ${this.context.turnCredentials.username}`);
```

Comment:
maybe add to same log line so we can easily grep them

Reply:
Yeah, good call. Updated to be in the same log line.